apps/ckit-app/src/hooks/useCurrentWalletStorage.tsx

```diff
@@ -0,0 +1,5 @@
+import { useLocalStorage } from '@rehooks/local-storage';
+
+export function useCurrentWalletStorage(): [string | null, (newValue: string | null) => void, () => void] {
+  return useLocalStorage<string | null>('currentWalletIndex');
```

Comment:
I found that the `currentWalletIndex` is actually  `currentWalletName`, and the hook can actually be placed under WalletContainer, can't it?

Reply:
yeah, and fixed